bin/ci-stack.ts

```diff
@@ -50,6 +50,13 @@ if (stage === 'Dev') {
     templateName: 'builder-template',
   };
 
+  const oscarBuildAccess: FineGrainedAccessSpecs = {
+    users: ['opensearch-ci-bot'],
+    roleName: 'oscar-build-role',
+    pattern: '(docker-scan|central-release-promotion)',
+    templateName: 'builder-template',
+  };
+
```

Comment:
You cann add pattern to the existing spec here https://github.com/opensearch-project/opensearch-ci/pull/611/files#diff-26e9372758e52b3d163810306a5dd2e6c26135b53454dad5ea7c4581129aff35R46-R51
Since the user is same, it should be fine. That was the intend of creating it in the first place.